data_parser/utils.py

```diff
@@ -66,6 +66,9 @@ def convert_pascal_camel_case_to_title_case(s: str):
 
 def convert_name_to_title_case(obj):
     if "name" not in obj:
+        if "Name" in obj:
+            obj["Name"] = convert_pascal_camel_case_to_title_case(obj["Name"])
+            return obj
         return obj
```

Comment:
I think you don't have to do this if you first run the `set_camel_case_keys`  on the mount object

Reply:
turns out i really only wanted to do `set_camel_case_keys`  I dont need to call `convert_to_title_case` so I removed these changes